doc/TESTING.md

```diff
@@ -16,23 +16,23 @@ The tests take between 6-12 minutes to run (we're working on shortening this!) s
 
 ## Coverage
 
-See [plots2 on CodeClimate](https://codeclimate.com/github/publiclab/plots2) for how well covered our code is with tests; we are extremely interested in building our out test suite, so please consider helping us write tests! 
+See [plots2 on CodeClimate](https://codeclimate.com/github/publiclab/plots2) for how well covered our code is with tests; we are extremely interested in building our out test suite, so please consider helping us write tests!
 
 ## Client-side tests
 
 Client-side tests (for JavaScript functions) are run using [Jasmine](https://jasmine.github.io/) in [jasmine-rails](https://github.com/searls/jasmine-rails). You can run tests by navigating to `/specs/` in the browser. Headless, or command-line test running may be possible with:
 
 `RAILS_ENV=test bundle exec rake spec:javascript`
 
-...[if you have phantomjs installed](#phantomjs-for-javascript-tests) (see above). 
+...[if you have phantomjs installed](#phantomjs-for-javascript-tests) (see above).
 
 ## Solr tests
 
-Solr (search) tests (generally not recommended!) require [installing the Solr search engine](#solr-search-engine) (see above). Once you've done that, you still need to turn it off in development mode before running tests, with `rake sunspot:solr:stop`. Read more about [this issue here](https://github.com/publiclab/plots2/issues/832#issuecomment-249695309). 
+Solr (search) tests (generally not recommended!) require [installing the Solr search engine](#solr-search-engine) (see above). Once you've done that, you still need to turn it off in development mode before running tests, with `rake sunspot:solr:stop`. Read more about [this issue here](https://github.com/publiclab/plots2/issues/832#issuecomment-249695309).
```

Comment:
We can remove this whole section on Solr - as we've stopped using it! Thank you!!!!